### PR TITLE
Spotty + Docker

### PR DESCRIPTION
**Pull request by turian**
This PR also includes tag v2021.0.1, which I pushed to pypi.

This PR creates tar-files at the end of luigi.

### CODE REVIEW

**Review by khumairraj — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -93,7 +100,7 @@ the label metadata for them.
 * SubcorpusSplit (subsample each split) => MonoWavTrimCorpus => SubcorpusData (symlinks)
 * SubcorpusData => {SubcorpusMetadata, ResampleSubcorpus}
 * SubcorpusMetadata => MetadataVocabulary
-* FinalizeCorpus
+* FinalCombine => FinalizeCorpus
```
Comment (jorshi) [inline README.md:103]: Add info about tar dir in args documentation below?

File: hearpreprocess/pipeline.py [hunk 2]
```diff
@@ -26,6 +28,9 @@
 TRAIN_PERCENTAGE = 100 - VALIDATION_PERCENTAGE - TEST_PERCENTAGE
 
 # We want no more than 5 hours of audio per task.
+# This can be overriden in the task config.
+# e.g. speech_commands test set.
+# If None, no limit is used.
```
Comment (jorshi) [inline hearpreprocess/pipeline.py:33]: Maybe include an example on how to do that, or least what key to use in task config?

File: hearpreprocess/pipeline.py [hunk 3]
```diff
@@ -870,6 +890,55 @@ def run(self):
         self.mark_complete()
 
 
+class FinalizeCorpus(MetadataTask):
```
Comment (jorshi) [inline hearpreprocess/pipeline.py:893]: What's the benefit of including this tar step in the pipeline? Won't we almost all the time be running this on like GCP and then using the generated files? Or are you imagining just running this once per tasks to create the tar and then uploading that for eval?

Comment (jorshi) [reply hearpreprocess/pipeline.py:893]: Are we going to host the final task tars on hugging face?

**Review by jorshi — APPROVED** [review]

File: hearpreprocess/pipeline.py [hunk 4]
```diff
@@ -870,6 +890,55 @@ def run(self):
...
+        return {
+            "combined": FinalCombine(
+                sample_rates=self.sample_rates,
+                tasks_dir=self.tasks_dir,
+                metadata_task=self.metadata_task,
+                task_config=self.task_config,
+            )
+        }
+
+    def create_tar(self, sample_rate: str):
```
Comment (jorshi) [inline hearpreprocess/pipeline.py:923]: This looks like all the sample rates end up in each of tars that are created (so we just end up with four tars that are exactly the same). It's also quite slow (quite a bit slower than the command line version https://stackoverflow.com/a/45621522) -- even for the small datasets this took several minutes. Maybe add a tqdm bar?

Comment (turian) [reply hearpreprocess/pipeline.py:923]: https://github.com/neuralaudio/hear-preprocess/issues/39

File: hearpreprocess/pipeline.py (hunk 3 above)
Comment (turian) [reply hearpreprocess/pipeline.py:893]: I don't want to upload with the script. Too easy to accidentally publish secret tasks. Must do it by hand.

**Pull request merged**